Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ruby:2.3.1
+RUN apt-get update -qq && apt-get upgrade -y
+
+RUN apt-get install -y build-essential nodejs && apt-get clean
```

Comment:
What environment is this run in, what version of Node.js will be installed?

Reply:
Whatever the base `ruby:2.3.1` image uses. I'll investigate.

Reply:
From what I can gather the Ruby image is built from Debian Jessie, which should have the later versions of Node.js.